### ISSUE TITLE
Differentiate various (from R6) identities

### ISSUE THREAD

**Issue by castaway** [runbox/runbox7#965]
![image](https://user-images.githubusercontent.com/109990/115544850-0310df00-a29b-11eb-8349-f13399c00562.png)

This is confusing, also the "select Default Identity" list includes "castaway@runbox.com" 3 times with no clue which is which.

After some discussion with @davidbowdley  this multitude turns out to be from various folder based idents created in R6 - we need a way to show what they are.

**Comment by gtandersen** [comment]
Agreed, or we automatically remove duplicates when importing from Runbox 6.

**Comment by hernan604** [comment]
It is possible to check if the identity is associated with a "folder" and hide it in rmm7.

**Comment by gtandersen** [comment]
Sounds good @hernan604 -- what do you think @runboxdave?

**Comment by runboxdave** [comment]
Well, it's a bit more complex than it seems because although @castaway 's identities all look the same technically they aren't because they are associated with different folders and sometimes customers use the same email address but with different signatures (and in theory they could also be using different reply-to addresses). 

So I would go with @castaway 's original point that we need to show what they are rather than deduplicate them (unless we really want to go to all the trouble to write code to do that for the ones that really are duplicates - maybe we should do that anyway).

So, given that each Identity has a "Description" field we could put something in there to say what it is e.g. "Profile from Inbox folder in RMM6. The use of the word profile is deliberate there because a) that's what we call then in RMM6 and b) it distinguishes them from the new Identities in Runbox 7.

However, we really should show that description field in the overview of the identities and not just when you edit one. I think that's been mentioned before as it is rather important in identifying them.

**Comment by gtandersen** [comment]
Very well, let's do what @runboxdave suggests and keep all the imported profiles and also add the Description field in the overview.

**Comment by castaway** [comment]
"Description" seems to be generally already set (on the ones I'm looking at), to the "name" field, so I'm adding an "Origin" one..

![image](https://user-images.githubusercontent.com/109990/136189990-e83a50b2-b3bd-408e-a2c9-c27935728056.png)

![image](https://user-images.githubusercontent.com/109990/136190031-44924794-c87c-493a-9b3b-430876ad3d47.png)

How's that?

**Comment by castaway** [comment]
I also made the "select default identity" list the "nameandaddress" so that we can differentiate the ones with the same email: 

![image](https://user-images.githubusercontent.com/109990/136191278-dde3aec0-8542-4d3a-852f-6bc990db2c1e.png)

**Comment by castaway** [comment]
Do you mean "don't show it for aliases" (which are kinda obvious), or for runbox7 identities.. did you still want the rmm6 folder name, or just "Runbox 6"?
